decidim-api/lib/decidim/api/engine.rb

```diff
@@ -23,11 +23,17 @@ module Api
     class Engine < ::Rails::Engine
       isolate_namespace Decidim::Api
 
+      initializer "decidim_api.configure" do |app|
+        # Remove the default middleware as we configure Warden::JWTAuth manually
+        # below, see `after_initialize`.
+        app.initializers.find { |a| a.name == "devise-jwt-middleware" }.context_class.instance.initializers.reject! { |a| a.name == "devise-jwt-middleware" }
+      end
+
```

Comment:
This does not do anything. The initializer is applied and run even when this code is applied.

Also, the initializer is needed because otherwise the JWT authentication would not work.

```suggestion
```